setup/action.yml

```
@@ -0,0 +1,56 @@
+name: Setup
+description: "Set up the Release Environment"
+inputs:
+  aws_role_arn:
+    description: "The aws role to acquire"
+    required: true
+  aws_region_name:
+    description: "The aws region to use"
+    required: true
+  aws_secret_id:
+    description: "The name of the aws secret to use"
+    required: true
+  app_id:
+    description: "The GitHub App id"
+    required: true
+  private_key:
+    description: "The GitHub App private key"
+    required: true
+  artifactory_registry:
+    description: "Artifactory registry to be used"
+    default: artifactory.corp.mongodb.com
+  artifactory_image:
+    description: "Image to use for artifactory"
+    default: release-tools-container-registry-local/garasign-git
+
```

This action should expose the generated token as an action output:
```suggestion
outputs:
  token:
    description: "GitHub installation access token"
    value: ${{ steps.app-token.outputs.token }}

```

This allows re-using the generated token for other steps.

---

Noted that the app-token step was removed 👍